gui: add git hash to version tooltip (#4187)
firedancer-io/firedancer: book/api/websocket.md

known cluster) when the validator learns its cluster from a downloaded
snapshot.

> #### `summary.git_hash`

Reviewer: `summary.commit_hash`

I wouldn't tie this to git specifically, in case it ever changes.

| frequency | type     | example         |
|-----------|----------|-----------------|
| *Once*    | `string` | `"78eefec7c779ef138aaaf4afe76cd6eaf4807006"` |

> The hash of the checked-out git commit at the time the validator was

Reviewer: ```suggestion
The commit hash used to build the validator.
```
